### PR TITLE
added free resources

### PR DESCRIPTION
**Pull request by SahityaRoy**
Hey Project admin @MRslimcoder sir can you please marge the pr i added some free resources

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ complete beginners.
 ### 2. Video Courses - Here are 2 from our range :
```
Comment (CodingSpecies) [inline README.md:126]: @SahityaRoy , as you added a link, can you please change the `Here are 2 from our range` to `Here are 3 from our range`. Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Comment by SahityaRoy** [comment]
i don't fet your point sir can u elaborate

File: README.md (hunk 1 above)
Comment (MRslimcoder) [reply README.md:126]: @SahityaRoy add 3 in description instead of two.

File: README.md [hunk 2]
```diff
@@ -171,6 +172,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
   <tr>
     <td align="center"><a href="http://carlosvinimsouza.github.io/"><img src="https://avatars.githubusercontent.com/u/60679101?v=4?s=100" width="100px;" alt=""/><br /><sub><b>CarlosViniMSouza</b></sub></a><br /><a href="#content-CarlosViniMSouza" title="Content">🖋</a></td>
   </tr>
+
```
Comment (MRslimcoder) [inline README.md:175]: Remove this tab from.

**Pull request closed without merging**